src/Workspaces/Core/Portable/FindSymbols/TopLevelSyntaxTree/IDeclaredSymbolInfoFactoryService.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Threading;
+using Microsoft.CodeAnalysis.Host;
+using Microsoft.CodeAnalysis.PooledObjects;
+
+namespace Microsoft.CodeAnalysis.FindSymbols
+{
+    internal interface IDeclaredSymbolInfoFactoryService : ILanguageService
```

Comment:
❓ why did this move?

Reply:
it was in the wrong folder and iw as very confused as i needed to step through this api :)